analysis_module/mockserver.py

```diff
@@ -0,0 +1,293 @@
+from typing import Any, List, Tuple
+import os
+import json
+import requests
+import logging
+import random
+import numpy as np
+from random import shuffle
+from math import ceil
+from celery import shared_task
+from sklearn.feature_extraction.text import CountVectorizer
+
+from analysis_module.models import AnalysisModuleRequest
+from core_server.settings import ENDPOINT_NAME
+from .utils import send_callback_url_request
+
+logging.getLogger().setLevel(logging.INFO)
+
+MOCK_GEOLOCATION: List = [
+    
+    {'ent': 'Cauca',
+    'offset_start': 0,
+    'offset_end': 0,
+    'geoids': [{'match': 'Departamento del Cauca',
+        'geonameid': 3687029,
+        'latitude': 2.5,
+        'longitude': -76.83333,
+        'featurecode': 'ADM1',
+        'contrycode': 'CO'}]},
+    {'ent': 'Amazonas',
+    'offset_start': 0,
+    'offset_end': 0,
+    'geoids': [{'match': 'Amazonas',
+        'geonameid': 3689982,
+        'latitude': -1.16667,
+        'longitude': -71.5,
+        'featurecode': 'ADM1',
+        'contrycode': 'CO'}]},
+    {'ent': 'Huila',
+    'offset_start': 0,
+    'offset_end': 0,
+    'geoids': [{'match': 'Departamento del Huila',
+        'geonameid': 3680692,
+        'latitude': 2.5,
+        'longitude': -75.58333,
+        'featurecode': 'ADM1',
+        'contrycode': 'CO'}]},
+    {'ent': 'Putumayo',
+    'offset_start': 0,
+    'offset_end': 0,
+    'geoids': [{'match': 'Departamento del Putumayo',
+        'geonameid': 3671178,
+        'latitude': 0.5,
+        'longitude': -76.0,
+        'featurecode': 'ADM1',
+        'contrycode': 'CO'}]}
+]
```

Comment:
@aembryonic : is this the final list that is received ? If so, there seems to be missing the entry_id unless you handle it differently. Please check the analysis module api document for the response structure.

Reply:
i know. it's not the final one.